cli/src/commands/branch/create.rs

```diff
@@ -25,7 +25,10 @@ use crate::ui::Ui;
 #[derive(clap::Args, Clone, Debug)]
 pub struct BranchCreateArgs {
     /// The branch's target revision
-    #[arg(long, short)]
+    ///
+    /// You can also use `--to` instead of `-r` for ease of switching between
+    /// `branch create`, `branch move`, and `branch set`.
```

Comment:
nit: I wouldn't add this because it's visible alias and user can find it in `branch create --help`.

Reply:
I felt like `create --to` deserves an explanation/excuse for why we'd go against the rules of English in this way.

Reply:
make it non-doc comment then? I agree it's informative for developers.

Reply:
OK, let's do that.